src/vl-map-search.js

```diff
@@ -260,7 +260,7 @@ class VlMapSearch extends vlElement(HTMLElement) {
 
   _changeTranslations() {
     this.placeholder = 'Lokaliseer adres';
-    this.searchPlaceholder = 'Gemeente, straat en huisnummer of Lambert-coördinaat';
+    this.searchPlaceholder = 'Zoeken op adres of coördinaat';
```

Comment:
even ter info. Mijn uitgangspunt om hier expliciet Lambert-coördinaat te zetten was omdat er meerdere coördinatenstelsels bestaan (en wij hier enkel Lambert aanvaarden). Nu ik weet niet of er zelfs andere coördinaten buiten Lambert gebruikt worden binnen de afnemers van deze component (of in de toekomst). Want indien dat de enige is kan je het hier idd gewoon korter schrijven.

Reply:
Gezien onze business en context kunnen we voorzichtig aannemen dat voor de gebruikers die coördinaten wensen te gebruiken de default van Lambert "evident" is. We kunnen dit natuurlijk altijd verder verfijnen als blijkt dat er toch verwarring zou zijn. Er kan ook steeds applicatief verduidelijkt worden welke en hoe de coördinaten gebruikt kunnen worden.